src/ResourceManager/Websites/Commands.Websites/help/Edit-AzureRmWebAppBackupConfiguration.md

```diff
@@ -0,0 +1,220 @@
+---
+external help file: Microsoft.Azure.Commands.Websites.dll-Help.xml
+ms.assetid: BFC38930-DBB4-4EBB-8E29-73B901FAF486
+online version: 
+schema: 2.0.0
+---
+
+# Edit-AzureRmWebAppBackupConfiguration
+
+## SYNOPSIS
+
+## SYNTAX
+
+### FromResourceName
+```
+Edit-AzureRmWebAppBackupConfiguration [-FrequencyInterval] <Int32> [-FrequencyUnit] <String>
+ [-RetentionPeriodInDays] <Int32> [[-StartTime] <DateTime>] [-KeepAtLeastOneBackup]
+ [-ResourceGroupName] <String> [-Name] <String> [[-Slot] <String>] [-StorageAccountUrl] <String>
+ [[-Databases] <DatabaseBackupSetting[]>] [<CommonParameters>]
+```
+
+### FromWebApp
+```
+Edit-AzureRmWebAppBackupConfiguration [-FrequencyInterval] <Int32> [-FrequencyUnit] <String>
+ [-RetentionPeriodInDays] <Int32> [[-StartTime] <DateTime>] [-KeepAtLeastOneBackup] [-WebApp] <Site>
+ [-StorageAccountUrl] <String> [[-Databases] <DatabaseBackupSetting[]>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Edit-AzureRmWebAppBackupConfiguration** cmdlet edits the current configuration backup for an Azure Web App.
+
+## EXAMPLES
+
+### 1:
```

Comment:
If you don't have time for examples, please remove the example (the ###1: and the ps c:\>text afterward).